### PR TITLE
CB-4095. Added env variables for configurations parameters

### PR DESCRIPTION
**Pull request by DenisSinelnikov**

### CODE REVIEW

File: config/sample-databases/DefaultConfiguration/cloudbeaver.conf [hunk 1]
```diff
@@ -27,11 +27,11 @@
...
         },
 
-        expireSessionAfterPeriod: 1800000,
+        expireSessionAfterPeriod: "${CLOUDBEAVER_EXPIRE_SESSION_AFTER_PERIOD:1800000}",
 
-        develMode: false,
+        develMode: "${CLOUDBEAVER_DEVEL_MODE:false}",
 
-        enableSecurityManager: false,
+        enableSecurityManager: "${CLOUDBEAVER_ENABLE_SECURITY_MANAGER:false}",
```
Comment (alexander-skoblikov) [inline config/sample-databases/DefaultConfiguration/cloudbeaver.conf]: we don't need a variable for this parameter right now

File: config/sample-databases/DefaultConfiguration/cloudbeaver.conf [hunk 2]
```diff
@@ -64,29 +64,29 @@
...
     },
     app: {
-        anonymousAccessEnabled: true,
-        anonymousUserRole: "user",
-        defaultUserTeam: "user",
-        grantConnectionsAccessToAnonymousTeam: false,
-        supportsCustomConnections: false,
-        showReadOnlyConnectionInfo: false,
+        anonymousAccessEnabled: "${CLOUDBEAVER_ANONYMOUS_ACCESS_ENABLED:true}",
+        anonymousUserRole: "${CLOUDBEAVER_ANONYMOUS_USER_ROLE:user}",
```
Comment (alexander-skoblikov) [inline config/sample-databases/DefaultConfiguration/cloudbeaver.conf]: anonymousUserRole is deprecated, we should use anonymousUserTeam

File: config/sample-databases/SQLiteConfiguration/cloudbeaver.conf [hunk 3]
```diff
@@ -61,28 +61,28 @@
 
     },
     app: {
-        anonymousAccessEnabled: true,
-        anonymousUserRole: "user",
-        grantConnectionsAccessToAnonymousTeam: false,
-        supportsCustomConnections: false,
-        showReadOnlyConnectionInfo: false,
+        anonymousAccessEnabled: "${CLOUDBEAVER_ANONYMOUS_ACCESS_ENABLED:true}",
+        anonymousUserRole: "${CLOUDBEAVER_ANONYMOUS_USER_ROLE:user}",
```
Comment (alexander-skoblikov) [inline config/sample-databases/SQLiteConfiguration/cloudbeaver.conf]: same here

File: config/sample-databases/DefaultConfiguration/cloudbeaver.conf [hunk 4]
```diff
@@ -64,29 +64,29 @@
...
 
         resourceQuotas: {
-            dataExportFileSizeLimit: 10000000,
-            resourceManagerFileSizeLimit: 500000,
-            sqlMaxRunningQueries: 100,
-            sqlResultSetRowsLimit: 100000,
-            sqlResultSetMemoryLimit: 2000000,
-            sqlTextPreviewMaxLength: 4096,
-            sqlBinaryPreviewMaxLength: 261120
+            dataExportFileSizeLimit: "${CLOUDBEAVER_DATA_EXPORT_FILE_SIZE_LIMIT:10000000}",
```
Comment (serge-rider) [inline config/sample-databases/DefaultConfiguration/cloudbeaver.conf]: LEt's name such variables with prefix. CLOUDBEAVER_RESOURCE_QUOTA_

File: config/sample-databases/SQLiteConfiguration/cloudbeaver.conf [hunk 5]
```diff
@@ -61,28 +61,28 @@
 
     },
     app: {
-        anonymousAccessEnabled: true,
-        anonymousUserRole: "user",
-        grantConnectionsAccessToAnonymousTeam: false,
-        supportsCustomConnections: false,
-        showReadOnlyConnectionInfo: false,
+        anonymousAccessEnabled: "${CLOUDBEAVER_ANONYMOUS_ACCESS_ENABLED:true}",
```
Comment (serge-rider) [inline config/sample-databases/SQLiteConfiguration/cloudbeaver.conf]: Var prefix CLOUDBEAVER_APP_

**Review by serge-rider — CHANGES_REQUESTED** [review]

**Review by alexander-skoblikov — APPROVED** [review]

**Review by serge-rider — APPROVED** [review]

**Comment by GenLN** [comment]
Those environment variables, can they be disabled? As our custom helm chart deployment with templated connections and configs wont't work if we don't overwrite cloudbeaver.conf in /opt/cloudbeaver location, as image load default one with those ENVs... Image 24.2.0 works great but this patch wouldn't start properly...

Either make all config setup to use ENVs for it, or leave the option to disable/enable those ENVs from image?

**Pull request merged**